src/core/views/layers/LayerOptionsModalView.js

```diff
@@ -31,31 +33,38 @@ const LayerOptionsModalView = ModalView.extend(/** @lends core/views/layers.Laye
 
   getDisplayOptions() {
     const display = this.useDetailsDisplay ? this.model.get('detailsDisplay') : this.model.get('display');
-    const options = display.options
-      .map((option) => {
-        let values = option.values;
-        let low;
-        let high;
-        let targetLow;
-        let targetHigh;
-        const target = option.target;
-        if (values) {
-          values = values.map(value =>
-            Object.assign({}, value, {
-              isCurrent: this.model.get(target) === value.value
-            })
-          );
-        }
-        if (typeof option.min !== 'undefined') {
-          [targetLow, targetHigh] = Array.isArray(target) ? target : target.split(',');
-          low = this.model.get(targetLow);
-          high = this.model.get(targetHigh);
-        }
-        return Object.assign({}, option, { values, low, high, targetLow, targetHigh });
-      });
+    if (display.options ==! undefined){
```

Comment:
This should propably be `!==` instead of `==!`

Also a check for `undefined` should be done like this: `if (typeof display.options !== 'undefined') {`